testing_app/test/home_test.dart

```diff
@@ -0,0 +1,69 @@
+// Copyright 2020 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:provider/provider.dart';
+import 'package:testing_app/models/favorites.dart';
+import 'package:testing_app/screens/home.dart';
+
+Widget createHomeScreen() => ChangeNotifierProvider<Favorites>(
+      create: (context) => Favorites(),
```

Comment:
Nit: 2-space indentation.

Reply:
The `flutter format` command formats it in this way. =)